docs/contributing.md

```diff
@@ -219,46 +219,67 @@ multi-threaded. It should use all the CPU cores without any configuration.
 
 ## Previewing the HTML documentation
 
-The documentation for `jj` is automatically published to the website at
+The documentation for `jj` is automatically published online at
 <https://martinvonz.github.io/jj/>.
 
-When editing documentation, we'd appreciate it if you checked that the
-result will look as expected when published to the website.
+When editing documentation, you should check your changes locally — especially
+if you are adding a new page, or doing a major rewrite.
 
-### Setting up the prerequisites
+The only thing you need is [`uv`][uv], a Python project manager written in Rust.
+It will fetch the right Python version and the dependencies. Install it like so:
 
-To build the website, you must have `uv` installed. Please see: [Installing uv].
+[uv]: https://docs.astral.sh/uv/
 
-[Installing uv]: https://docs.astral.sh/uv/getting-started/installation/
+=== "macOS/Linux"
 
-Once you have `uv` installed, you should ask it to install the rest
-of the required tools into a virtual environment as follows:
+    ``` { .shell .copy }
+    curl -LsSf https://astral.sh/uv/install.sh | sh
```

Comment:
One annoyance I have about `uv` installer is that is modifies you shell RC files by default.

I'm not sure, but I wonder whether we should do:

```shell
curl -LsSf https://astral.sh/uv/install.sh | INSTALLER_NO_MODIFY_PATH=1 sh
# Or, if ~/.local/bin is not already in your path
curl -LsSf https://astral.sh/uv/install.sh | sh
```

WDYT?

Reply:
I read the installer and it seems to skip the PATH modification if the dir is already in the path 

This said, some distros (Arch) don’t have it in path, so I can add a note that uv will be installed into ~/.local/bin and auto-added to path

Reply:
I find that whenever I don't add that env var, it creates `~/.local/env` file and maybe `env.fish`, and adds it to my `.bashrc`. Maybe it's just me.

I'm not sure, but it might also want to add the "tool bin" directory to the PATH, which is not needed for us. (Though it could mean that people using our instructions might be surprised it's not in the PATH)

Reply:
<img width="798" alt="image" src="https://github.com/user-attachments/assets/3b67486a-335e-4e69-8d67-63d136783f58">

Reply:
Thank you!